low-level/files/opt/seagate/sspl/bin/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -311,12 +312,12 @@ if __name__ == "__main__":
             print("Unrecognized arguments: %s" % sys.argv)
             sys.exit(1)
         tmpFile = open(onGoingRequestPath, "w")
-        os.makedirs(f"/tmp/{sspl_bundle_tmp}")
+        os.makedirs(sspl_bundle_tmp)
```

Comment:
We can also cleanup the `sspl_bundle_tmp` if it already exists. If any of below three steps fails for some reason, we may have left over files.

Reply:
Done